src/backend/distributed/sql/citus--8.3-1--9.0-1.sql

```diff
@@ -21,13 +21,7 @@ ALTER FUNCTION citus.restore_isolation_tester_func SET SCHEMA citus_internal;
 GRANT USAGE ON SCHEMA citus TO public;
 
 #include "udfs/pg_dist_shard_placement_trigger_func/9.0-1.sql"
-
-CREATE OR REPLACE FUNCTION pg_catalog.worker_create_or_replace_object(statement text)
-  RETURNS bool
-  LANGUAGE C STRICT
-  AS 'MODULE_PATHNAME', $$worker_create_or_replace_object$$;
-COMMENT ON FUNCTION pg_catalog.worker_create_or_replace_object(statement text)
-    IS 'takes a sql CREATE statement, before executing the create it will check if an object with that name already exists and safely replaces that named object with the new object';
+#include "udfs/worker_create_or_replace_object/9.0-1.sql"
```

Comment:
We don't have to, as long as we are keeping the old function around.
The reason I changed the version from the past was mostly because this UDF wasn't nicely versioned which made it harder to change the function.

If we decide to keep this function now we can do without the 9.0-1 file. However if we decide to _only_ take an array we will require the 9.0-1 file for the downgrade path. If we have the 9.0-1 file I think it is better to use it here as well.

Basically when the function got introduced it should have created this file (and latest.sql in the same dir) to make versioning in the future easier.

Reply:
With using the `link_name` for the binding of C functions to the sql UDFs I have renamed the original C function to reflect the `_text` argument. Hence it is easiest now to have the `9.0-1.sql` file which was missing before.